src/main/kotlin/org/sert2521/infiniterecharge2020/powerhub/PowerhubCommands.kt

```diff
@@ -0,0 +1,28 @@
+package org.sert2521.infiniterecharge2020.powerhub
+
+import org.sert2521.sertain.coroutines.doAll
+import org.sert2521.sertain.coroutines.periodic
+import org.sert2521.sertain.events.onTick
+import org.sert2521.sertain.subsystems.doTask
+import org.sert2521.sertain.subsystems.use
+
+suspend fun spin(intake: Boolean) = doTask {
+    val powerHub = use<PowerHub>()
+    action {
+        doAll {
+            onTick {
+                periodic(20) {
+                    powerHub.spin()
+                    if (intake) {
+                        println("Trying to intake")
+                        powerHub.lowerFlapper()
+
+                    } else {
+                        println("Trying to outtake")
+                        powerHub.raiseFlapper()
```

Comment:
I checked in with CAD again and apparently we don't need to set continuous power to the motor to keep the flap open, and closing the power will just require motor power in the opposite direction. Not sure if we _can_ still run the motor the whole time or if we should turn it off as soon as it opens. Can ask CAD/Mechanical more about that. If we do need to turn it off, we can probably just put it in a `timer` loop